.github/workflows/build.yml

```
@@ -41,7 +41,7 @@ jobs:
 
     - name: Upload Meson logs
       if: always()
-      uses: actions/upload-artifact@v4
+      uses: actions/
```

actionlint (1.7.12)

[error] specifying action "actions/" in invalid format because ref is missing. available formats are "{owner}/{repo}@{ref}" or "{owner}/{repo}/{path}@{ref}"

(action)